src/main.js

```diff
@@ -69,7 +69,18 @@ function waitForGuild(guildId) {
 
 function initStatus() {
   function applyStatus() {
-    bot.editStatus(null, {name: config.status});
+    let statusType = config.statusType;
+    if (typeof statusType === "string") {
+      statusType = parseInt(statusType);
+      if (isNaN(statusType))
```

Comment:
It should be possible to greatly simplify these conditionals.  If statusType === 0 || 2 || 3, editStatus to statusType, else editStatus to 0.  Two conditionals, as opposed to the 4 currently in use.  Not a huge deal, especially on something that only happens during startup, but just from principle.

Reply:
Good point, those if conditions definitely can be combined.